tests/requirements.txt

```diff
@@ -1,5 +1,6 @@
+codeclimate-test-reporter==0.2.1
 flake8==3.5.0
-pytest-cov==2.4.0
+mock==2.0.0
+libnfs==1.0.post4
```

Comment:
I don't think this dependency should be listed as a tests/requirements.txt, I think you want to add that to setup.py, right?

Reply:
Maybe? It will force users to install the libnfs ubuntu package regardless if they use this check... Not sure how I feel about that.